docs/developer/index.rst

```diff
@@ -1079,7 +1109,7 @@ The :py:meth:`sherpa.ui.utils.Session.plot` and
 plots to be created by specifying the plot type as a list of
 argumemts. For example::
 
-    >>> s.plot('data', 'model', 'data', 2, 'model', 2)
+    >>> s.plot('data', 'model', 'data', 2, 'model', 2)  # doctest: +SKIP
```

Comment:
Just to check: the reason for skipping this is not because we don't want to check the screen output here (which doesn't exist, but you might want to enforce this), but because this is a snippet where we haven't set up the requirements - i.e. we haven't loaded data or created models for the default id and id 2)?

That is, we have multiple reasons to skip a test.

I'm also pleased to see that the doctest comment is skipped in the output.

Reply:
Yes and yes. (We did not set it up and I don't want to compare the screen output). And yes, with the plugin activates, Sphinx knows about doctest and will remove that markup. The question is if there is any place where we have to worry about that for other downstream code, e.g. do we make ahelp files from rst files or docstrings?

Reply:
The ahelp files are generated from the sherpa.astro.ui.utils docstrings (and hence sherpa.ui.utils). I'd have to check how the rst code represents such docstrings (ie how to filter them out). I think though that it would take time to get to these docstrings, as they are the most complex to do (as they hit lots of the general issues I'm talking about here as I try to understand the scope).